test/common/event/dispatcher_impl_test.cc

```diff
@@ -8,12 +8,17 @@
 #include "common/stats/isolated_store_impl.h"
```

Comment:
Where did we end up with the "should new tests be written?" thread?

Reply:
We ended up with me dragging my heels. I have the one new unit test here, but I guess it could be worthwhile to add an integration test somewhere, asserting that the stats we expect to populate are actually there...

Reply:
Yeah, this could easily be added to something like `server_test` or one of the integration test. Verify that we have the server and worker histogram stats available. You could also consider posting an event to one of the dispatchers that sleeps for 100ms (real time, not simulated, since that's what libevent uses) and verifying that the epoll duration is at least 100ms.

Reply:
Well, my first attempt at adding something to `server_test` was a non-starter: see #6582 for details. I'll try adding to an integration test instead, I may have better luck with that.

Reply:
... No luck there either, same issue: integration tests use `IsolatedStoreImpl` which doesn't really understand histograms. @htuch, would you mind if I punt on this until we have a fix for #6582?

Reply:
Yeah, this is fine, but please do track as I think we have some functional validation missing.